docs/install/Knative-with-any-k8s.md

```diff
@@ -10,11 +10,9 @@ using pre-built images.
 
 ## Before you begin
 
-Knative requires a Kubernetes cluster v1.11 or newer with the
-[MutatingAdmissionWebhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-controller)
-enabled. `kubectl` v1.10 is also required. This guide assumes that you've
-already created a Kubernetes cluster which you're comfortable installing _alpha_
-software on.
+Knative requires a Kubernetes cluster v1.14 or newer. `kubectl` v1.10 is also
```

Comment:
@mattmoor  Should we also bump up the required kubectl version?

Reply:
I don't know what informs our requirements there, so 🤷‍♂